webcrypto/tests/wpt-patches/WebCryptoAPI__derive_bits_keys__ecdh_bits.js.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/WebCryptoAPI/derive_bits_keys/ecdh_bits.js b/WebCryptoAPI/derive_bits_keys/ecdh_bits.js
+index 36b29c20a..e3bd4ccb5 100644
+--- a/WebCryptoAPI/derive_bits_keys/ecdh_bits.js
++++ b/WebCryptoAPI/derive_bits_keys/ecdh_bits.js
+@@ -1,7 +1,7 @@
+ 
+ function define_tests() {
+     // May want to test prefixed implementations.
+-    var subtle = self.crypto.subtle;
++    var subtle = crypto.subtle;
```

Comment:
Again I comment on this in the RSA test , but we can set `globalThis.self` to `globalThis` and won't need to do this everywhere.